uuid/uuid.go

```diff
@@ -27,25 +27,6 @@ func IsAndroid(uuid string) bool {
 	return IsValidFast(uuid) && matchUuidRegexAndroid(uuid) && !isTest(uuid)
 }
 
-type IdentityType int
-
-const (
-	IdentityTypeOther IdentityType = iota
-	IdentityTypeIDFAOrAAID
-	IdentityTypeNoID
-)
-
-func GetIdentity(uuid string) IdentityType {
-	if IsNoID(uuid) {
-		return IdentityTypeNoID
-	}
-	return IdentityTypeIDFAOrAAID
-}
-
-func IsNoID(uuid string) bool {
```

Comment:
Why did u remove IsNoID - we decided that we will leave it, right? https://remerge.slack.com/archives/C017ESKTRLY/p1595258249014900?thread_ts=1595257965.014400&cid=C017ESKTRLY

I think it makes sense to leave it, it is faster than getting the enum, and does not involve checking the uuids.

Sometimes u need a fast way to check it is NOID - and then it will work

Reply:
I removed it from here because I think it's still business logic. E.g. if I knew nothing about identity and found a package called `go-tools` with a method `IsNoID` I would likely assume this will tell me if something is a (valid) ID or not. zeroed out id is still valid, according to classic ID validation rules, it's only NoID due to our business logic and the agreed upon definition of NoID.

that said, I do think having the method is valuable, so WDYT about this solution instead? https://github.com/remerge/events-proto-go/pull/8/commits/f614f935ff4283e655c299f8e9db96c7c93918e5

Reply:
ok, I still disagree, because
* We have business logic here - `IsNoId` is the same as `IsAndroid` for example
* remerge/events-proto-go uses go-tools, so it can just reuse this package
* But again, this is all kind theoretical. If u want so - lets do so

Reply:
For me `IsAndroid` is an industry standard, and `IsNoID` is our internal business logic (at least for now).
lets keep it in events then, as that way we also have all identity related logic in one place